.github/workflows/build_test.yml

```
@@ -74,7 +74,7 @@ jobs:
     container: quay.io/pypa/manylinux2010_x86_64
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```

```suggestion
        python-version: [3.6, 3.7, 3.8, 3.9, "3.10"]
```
It seems that yaml parser treats `3.10` as float `3.1`, which has `"3.1"` string representation, which breaks the build. Don't forget to make the same changes further in this file.

It seems that `.github/workflows/create_req_files.yml` needs respective changes too.